pages/Settings.qml

```diff
@@ -358,57 +340,73 @@ Rectangle {
 
             LineEdit {
                 id: daemonPassword
-                Layout.preferredWidth: 100
+                Layout.preferredWidth: 100 * scaleRatio
                 Layout.fillWidth: true
                 text: persistentSettings.daemonPassword
                 placeholderText: qsTr("Password") + translationManager.emptyString
                 echoMode: TextInput.Password
             }
+        }
 
-            StandardButton {
-                id: daemonAddrSave
-                Layout.fillWidth: false
-                Layout.leftMargin: 30
-                text: qsTr("Connect") + translationManager.emptyString
-                shadowReleasedColor: "#FF4304"
-                shadowPressedColor: "#B32D00"
-                releasedColor: "#FF6C3C"
-                pressedColor: "#FF4304"
-                onClicked: {
-                    console.log("saving daemon adress settings")
-                    var newDaemon = daemonAddr.text.trim() + ":" + daemonPort.text.trim()
-                    if(persistentSettings.daemon_address != newDaemon) {
-                        persistentSettings.daemon_address = newDaemon
+        RowLayout {
+            visible: persistentSettings.useRemoteNode
+            ColumnLayout {
+                Label {
+                    color: "#4A4949"
+                    text: qsTr("Remote node") + translationManager.emptyString
+                }
+                RemoteNodeEdit {
+                    id: remoteNodeEdit
+                    Layout.minimumWidth: 100 * scaleRatio
+                    daemonAddrText: persistentSettings.remoteNodeAddress.split(":")[0]
+                    daemonPortText: (persistentSettings.remoteNodeAddress.split(":")[1] == "") ? "18081" : persistentSettings.remoteNodeAddress.split(":")[1]
```

Comment:
Maybe want to trim here, unless all writers to persistentSettings.remoteNodeAddress already take care of it.

Reply:
trimmed